tests/installation/addon_products_sle.pm

```diff
@@ -135,19 +135,12 @@ sub handle_addon {
 }
 
 sub test_addonurl {
-    my $testvalue = get_var('ADDONURL');
-    my @missing_modules;
-    my @test_modules = split(/,/, get_var('WORKAROUND_MODULES'));
+    die('ADDONURL is required') unless (get_var('ADDONURL') && check_var('MACHINE', '64bit-staging'));
```

Comment:
I'm not sure what is the logic behind this. I would recommend not to use `MACHINE` setting in the code, as if it gets renamed, code will break. If you want to limit this to staging tests only, you can use `is_staging()` method.